package.json

```diff
@@ -1,15 +1,48 @@
 {
-  "name": "code-exercise",
-  "version": "1.0.0",
-  "description": "This is a shopping cart coding exercise",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "canned -p 5000 ./server"
-  },
-  "author": "",
-  "license": "MIT",
-  "dependencies": {
-    "canned": "^0.3.12"
-  }
-}
+    "name": "shopping-app",
+    "version": "1.0.0",
+    "description": "This is a shopping cart coding exercise",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "compile:sass": "node-sass resources/sass/main.scss resources/css/style.css",
+        "start": "node ./bin/www",
+        "start:dev": "npm run compile:sass-dev"
+    },
+    "author": "Navin Tyagi",
+    "license": "MIT",
+    "devDependencies": {
+        "babel-preset-env": "^1.7.0",
+        "browser-sync-webpack-plugin": "^2.2.2",
+        "copy-webpack-plugin": "^5.0.2",
+        "css-loader": "^2.1.1",
+        "express": "^4.16.4",
+        "express-handlebars": "^3.0.2",
+        "extract-text-webpack-plugin": "^4.0.0-beta.0",
+        "file-loader": "^3.0.1",
+        "hbs": "^4.0.3",
+        "html-webpack-plugin": "^3.2.0",
```

Comment:
I can see quite a few node modules not in use. Still they are floating around in package.json

Reply:
done